src/main/resources/org/sagebionetworks/web/client/widget/entity/renderer/EntityListWidgetViewImpl.ui.xml

```diff
@@ -26,7 +26,7 @@
 						<t:TableHeader width="165px" addStyleNames="visible-md visible-lg">
 							<bh:Span text="Date" />
 						</t:TableHeader>
-						<t:TableHeader width="310px" addStyleNames="visible-xl">
+						<t:TableHeader width="310px" addStyleNames="visible-md visible-lg">
```

Comment:
Forgot we extended the bootstrap classes, visible-xl is our own media rule.  I'll take a look at what you have today...